.eslintignore

```diff
@@ -0,0 +1,2 @@
+spec
+app/models
```

Comment:
Please do not ignore the models. There is a way to disable eslint on a specific line or rule.